### PR TITLE
fix for E702 in pyx outside rings

### PR DESCRIPTION
**Pull request by fchapoton**
This fixes pycodestyle warning E702 in pyx files outside of rings

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/groups/perm_gps/partn_ref/refinement_binary.pyx [hunk 1]
```diff
@@ -468,8 +480,9 @@ cdef class NonlinearBinaryCodeStruct(BinaryCodeStruct):
         storing results to self.
 
         INPUT:
-        partition -- an optional list of lists partition of the columns.
-            default is the unit partition.
+
+        - partition -- an optional list of lists partition of the columns.
```
Comment (dcoudert) [inline src/sage/groups/perm_gps/partn_ref/refinement_binary.pyx]: partition -> ` ``partition`` `

File: src/sage/coding/binary_code.pyx [hunk 2]
```diff
@@ -3641,10 +3664,12 @@ cdef class BinaryCodeClassifier:
                 if tvc & nu.flag:
                     i = tvc^nu.flag
                     if Theta.wd_min_cell_rep[Theta.wd_find(i)] == i:
-                        state = 11; continue
+                        state = 11
+                        continue
                 else:
                     if Theta.col_min_cell_rep[Theta.col_find(tvc)] == tvc:
```
Comment (dcoudert) [inline src/sage/coding/binary_code.pyx]: `elif` ?

**Review by dcoudert — CHANGES_REQUESTED** [review]

File: src/sage/matrix/matrix2.pyx [hunk 3]
```diff
@@ -6018,7 +6020,8 @@ cdef class Matrix(Matrix1):
             sage: t.charpoly()                                                          # needs sage.libs.pari
             x^3 - 12*x^2 - 18*x
         """
-        i = int(i); t=int(t)
+        i = int(i)
+        t=int(t)
         if self.nrows() != self.ncols():
```
Comment (kwankyu) [inline src/sage/matrix/matrix2.pyx:6025]: Spaces around.

**Comment by kwankyu** [comment]
Otherwise lgtm.

**Review by kwankyu — APPROVED** [review]
Thanks.

**Pull request merged**